src/libstd/sys/unix/ext/net.rs

```diff
@@ -121,6 +136,21 @@ impl SocketAddr {
     }
 
     /// Returns true if and only if the address is unnamed.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::os::unix::net::{UnixDatagram, UnixListener};
+    ///
+    /// let socket = UnixListener::bind("/tmp/sock").unwrap();
+    /// let addr = socket.local_addr().expect("Couldn't get local address");
+    /// assert_eq!(addr.is_unnamed(), false);
+    ///
+    /// // with is_unnamed() as true now:
```

Comment:
Nit: Alternatively, you could also do something like this:

    # Examples

    An unnamed address:

    ```
    ...
    ```

    A named address:

    ```
    ...
    ```